web/src/components/njwds-extended/UnStyledButton.tsx

```diff
@@ -42,14 +32,12 @@ export const UnStyledButton = forwardRef(
       }
     }, [height, width, disabledClass]);
 
-    const noRightMargin =
```

Comment:
[boulder] This change affects the the Primary and Secondary buttons. Are the style changes affecting these other two components in a way that we want them displayed?

Reply:
Based on how the logic was written, the no right margin was applied to every button except the "help" formation button. I've corrected that padding.